### PR TITLE
Update requirements for install and docs

### PR DESCRIPTION
**Pull request by pomegranited**
Fixes document build process to work with readthedocs.org.  

**JIRA tickets**: [OLIVE-17](https://openedx.atlassian.net/browse/OLIVE-17), [OC-1392](https://tasks.opencraft.com/browse/OC-1392)

**Discussions**: Issues discovered during docs deployment, in discussion with @pdesjardins .

**Dependencies**: None

_Note:_ libpq-dev is no longer required to build the docs.

**Screenshots**:

[readthedocs :: Advanced Configuration](https://readthedocs.org/dashboard/edxopencraft-analytics-pipeline/advanced/)
readthedocs :: Advanced Configuration

[http://edxopencraft-analytics-pipeline.readthedocs.org/](thttp://edxopencraft-analytics-pipeline.readthedocs.org/en/latest/)
http://edxopencraft-analytics-pipeline.readthedocs.org/

**Sandbox URL**: 

http://edxopencraft-analytics-pipeline.readthedocs.org

**Testing instructions**:

On readthedocs.org:
1. Create a project using repository URL: https://github.com/edx/edx-analytics-pipeline
2. Use the default settings, with these changes to Advanced Settings:
   - Enable "Install your project inside a virtualenv using setup.py install" 
   - Requirements file: requirements/docs.txt
   - Default branch (for now): jill/make-docs-requirements
3. On the Overview page, click Build

On local machine:
1. Set up a virtualenv, e.g.:
   
   ```
   virtualenv analytics-docs
   source analytics-docs/bin/activate
   ```
2. Build the html docs (installs the pip dependencies, and the analytics pipeline service):
   
   ```
   make docs
   ```
3. Documentation is generated to:
   
   ```
   docs/* - HTML documentation
   docs/source/toctree.rst - table of contents
   docs/source/all.rst - all pipeline services
   docs/source/workflow_entry_point.rst - workflow entry points (main user docs)
   ```

**Reviewers**
- [x] @e-kolpakov  
- [ ] @mulby 
- [ ] @pdesjardins

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pomegranited! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?number=217&repo=edx%2Fedx-analytics-pipeline

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Your machine will need the following in order to run the code in this repository
 * [Python](https://www.python.org/) 2.7.x
 * [GCC](http://gcc.gnu.org/) (to compile numpy)
 * [MySQL](http://mysql.com)
+* [PostgreSQL](http://postgresql.com) (to write to Vertica)
```
Comment (bradenmacdonald) [inline README.md]: I don't think PostgreSQL is actually required - just the libpq PostgreSQL client library and its header files.

Comment (pomegranited) [inline README.md]: @bradenmacdonald [Stackoverflow agrees with you](http://stackoverflow.com/a/5450183) :)

Mac brew only has the whole postgresql, but any deb system should be able to install libpq-dev on its own.  I'll verify on my ubuntu VM.

[fed204c](https://github.com/edx/edx-analytics-pipeline/pull/217/commits/fed204cc4c79361e20673ad7151debbe8d616552) clarifies the postgres requirement to specify libpq-dev.  Is that sufficient information?

Comment (bradenmacdonald) [inline README.md]: Great!

**Comment by e-kolpakov** [comment]
@pomegranited :+1:

**Comment by pomegranited** [comment]
@mulby Could you please review this change?

**Comment by mulby** [comment]
:+1:

**Comment by pdesjardins** [comment]
Looks good, I'll test as soon as I can. Thanks!

**Comment by e-kolpakov** [comment]
@pomegranited still :+1:

**Pull request merged**